src/Legacy/Storage.php

```diff
@@ -1831,12 +1817,11 @@ protected function executeGetContentQueries($decoded)
      *
      * @param string $textquery
      * @param string $parameters
-     * @param array  $pager
      * @param array  $whereparameters
      *
      * @return array
      */
-    public function getContent($textquery, $parameters = '', &$pager = [], $whereparameters = [])
+    public function getContent($textquery, $parameters = '', $whereparameters = [])
     {
```

Comment:
Sure, but if someone does:

```php
$textquery = 'pages';
$parameters = 'foo';
$whereparameters = ['status' => 'hello'];

$content = $app['storage']->getContent($textquery, $parameters, [], $whereparameters);
```

They are going to lose their `$whereparameters`. One of many reasons to pass around objects and not arrays. Unless someone has a brilliant idea, I would think we'd need it for BC until we remove that class.